### PR TITLE
fix(Logs Kinesis Firehose): CF-10272

### PR DESCRIPTION
**Pull request by barbnewrelic**
Added clarification that optional metadata settings in AWS = attribute/value pairs in New Relic

### CODE REVIEW

**Comment by barbnewrelic** [comment]
Fixes https://github.com/newrelic/docs-website/issues/1524

**Comment by barbnewrelic** [comment]
@mculmone  As I noted in the Jira, I've assigned you as reviewer, but please feel free to add or change to other reviewers. Thanks!

**Review by arenjae — APPROVED** [review]

File: src/content/docs/logs/forward-logs/stream-logs-using-kinesis-data-firehose.mdx [hunk 1]
```diff
@@ -35,7 +35,9 @@ To forward your logs from Kinesis Data Firehose to New Relic:
   **Note**: To send your logs to the EU, complete the remaining steps in this section, then proceed to the [configuration procedures for EU accounts](#configure-eu-stream).
 
 7. Paste your [license key](/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key) in the **API key** field.
-8. Configure and review the remaining settings.
+8. Configure and review the remaining metadata settings.
+
+Any optional key/value pairs you add in the AWS Management Console will result in attribute/value pairs you can use in New Relic.
```
Comment (paperclypse) [inline src/content/docs/logs/forward-logs/stream-logs-using-kinesis-data-firehose.mdx:40]: I was a bit thrown by this sentence. How about something more like? 

```suggestion
Any optional key/value pairs you add to the AWS Management Console create attribute/value pairs you can use in New Relic.
```

**Review by paperclypse — APPROVED** [review]
Take or leave my suggestion. :)

**Pull request merged**